src/actions/performApiDataRequest.ts

```diff
@@ -93,11 +93,15 @@ export const performApiRequest: PerformApiRequest = (
             return Promise.reject(errorMsg);
         }
 
+        // Merge the defaultParams and URL params. This is where any defaultParams get overwritten.
+        // TODO: Bedenk betere naam voor endParams.
+        const endParams = { ...config.defaultParams, ...params };
```

Comment:
is het niet een idee om `endParams` te hernoemen naar `params` en de input params van de functie te hernoemen naar bijvoorbeeld `inputParams`?

Reply:
Goed punt, ik heb het aangepast.